ogr/abstract.py

```diff
@@ -93,7 +93,7 @@ def __str__(self) -> str:
         )
 
 
-class PRStatus(IntEnum):
+class PRFlag(IntEnum):
```

Comment:
Please, leave `PRStatus` as is and create a new Class for `PRFlag`.

status = open/close/merged
flag = CI status (only for PRs and commits)